internal/storage/compact/compact.go

```diff
@@ -134,7 +135,9 @@ func (s *Storage) Compact(ctx context.Context) (interface{}, error) {
 
 	// Wait for the pool to be close
 	close(queue)
-	return wpool.Outcome()
+	out, err := wpool.Outcome()
+	s.monitor.Histogram(ctxTag, "compactlatency", float64(time.Since(st)))
```

Comment:
I thought we had `monitor.Duration` or something

Reply:
We also need the number of times it was called. Duration just stores the time.